Orlando Walking Tours/Services/InMemoryModelService.swift

```diff
@@ -28,16 +28,35 @@ class InMemoryModelService : ModelService {
         self.addLocation(locations[5], toTour: cityTour, completion: nil)
     }
     
-    func findAllTours() -> NSArray {
+    func findAllTours() -> [Tour]? {
         return tours
     }
+
+    func findTour(byUUID uuid: NSUUID, completion: Tour? -> Void) {
+        for tour in tours {
+            if tour.uuid == uuid {
+                completion(tour)
+            }
+        }
+    }
     
-    func createTour(title title: String, completion: ModelServiceCompletionHandler?) {
+    func createTour(uuid uuid: NSUUID, title: String, completion: ModelServiceCompletionHandler?) {
         let newTour = Tour.MR_createEntity()!
+        newTour.uuid = uuid.UUIDString
         newTour.title = title
-        tours.addObject(newTour)
+        tours.append(newTour)
         completion?(true, nil)
     }
+
+    func deleteTour(tour tour: Tour, completion: ModelServiceCompletionHandler?) {
+        for localTour in tours {
```

Comment:
You know, I'm sort of thinking about just removing this entire file.  This was originally put in place because the model service that dealt with the actual persistent store wasn't ready.  Since I've been working on getting that service to work, this service never gets called at all.